README.md

```diff
@@ -13,10 +13,13 @@ Sky Walking | [中文](https://github.com/wu-sheng/sky-walking/wiki/sky-walking-
 [![OpenTracing-1.0 Badge](https://img.shields.io/badge/OpenTracing--1.0-enabled-blue.svg)](http://opentracing.io)
 
 
-* Auto-instrumentation mechanism, **no need to CHANGE any application source code**. [Supported middlewares, frameworks and libraries](https://github.com/wu-sheng/sky-walking/wiki/3.0-supported-list).
-* Explicit-instrumentation, as an [OpenTracing supported tracer](http://opentracing.io/documentation/pages/supported-tracers).
-* Pure Java server implementation, with RESTful Web service.
-* High performance streaming analysis.
+* Auto instrumentation java agent, **no need to CHANGE any application source code**. 
+  * [Supported middlewares, frameworks and libraries](https://github.com/wu-sheng/sky-walking/wiki/3.0-supported-list).
+* Manual instrumentation
+  * As an [OpenTracing supported tracer](http://opentracing.io/documentation/pages/supported-tracers)
+  * Use **@Trace** annotation for any methods you want to trace.
+* Pure Java server implementation, provide RESTful and gRPC services. Compatibility with other language agents/SDKs. 
+  * [How to uplink metrics and traces to collector?]()
```

Comment:
@pengys5 Before we release 3.2, we should provide a document about the work flows about **"How gRPC and RESTful service work"**